README.md

```diff
@@ -40,3 +40,6 @@ Number of days to query, with today as the end date. Defaults to 7 days.
 
 - `usernames`
 Your user handle, or your list of teammates, comma-separated.
+
+- `project_field`
+A project field to display for issues and pull requests.
```

Comment:
```suggestion
A project field to display for issues and pull requests. For example, "Epic" for relationships with "Epic" issues.
```

Is this right? It reminds me of the literal project board when you say "Project", but maybe I misunderstand how this query works.

Reply:
By `project field` I mean a (single select) field defined in a Project, like these (`Status` and `Type`):
![image](https://github.com/inkblotty/my-work-action/assets/44614829/026bbd32-fd59-417e-8954-34d62019be22)

What the query does, is basically the following:

1. For every issue and pull request:
2. For every project the issue/PR belongs to:
3. Extract the value of the `${project_field}` single select field (if such a field exists in the project)

Reply:
Ok that makes total sense. Let me try again.

```suggestion
A project field to display for issues and pull requests. This is usually a column (on Table views) from a GitHub Project, such as Status or Iteration.
```